src/js/profile/mobile/widget/Calendar.js

```diff
@@ -172,40 +218,90 @@
 				var calendarElement = this.element.querySelector(".calendarView");
 
 				if (event.srcElement.className === "calendarArrowRight") {
-					todayMonth = todayMonth + 1;
-					if (todayMonth === 13) {
-						todayMonth = 1;
-						todayYear = todayYear + 1;
-					}
-					prototype._deleteCalendar(calendarElement);
-					today = new Date(todayYear, todayMonth - 1);
-					prototype._buildCalendar(calendarElement);
+					prototype._moveMonth("right", calendarElement);
 				} else if (event.srcElement.className === "calendarArrowLeft") {
-					todayMonth = todayMonth - 1;
-					if (todayMonth === 0) {
-						todayMonth = 12;
-						todayYear = todayYear - 1;
-					}
-					prototype._deleteCalendar(calendarElement);
-					today = new Date(todayYear, todayMonth - 1);
-					prototype._buildCalendar(calendarElement);
+					prototype._moveMonth("left", calendarElement);
+				} else if (event.srcElement.className === "calendarArrowLeft nonSelectionArrow") {
+					return;
+				} else if (event.srcElement.className === "prevDisableDay nonSelectionDay" ||
+					event.srcElement.firstChild.className === "prevDisableDay nonSelectionDay") {
+					return;
+				} else if (event.srcElement.className === "prevDisableDay") {
+					prototype._moveMonth("left", calendarElement);
```

Comment:
use `this` or `self` instead of `prototype`

Reply:
done.